docs/requestAwareMiddleware.md

```diff
@@ -0,0 +1,45 @@
+# The "Request Aware" middleware.
+
+Some middleware (such as the "static" middleware included in this package) do not keep any kind of state variables that
```

Comment:
We agreed to clarify this that no state variables should be saved in middleware, and that this grade adds an additional level of protection to ensure that this can happen while still being able to use x... y... and z from the framework.